main.go

```diff
@@ -75,14 +75,20 @@ func generate(name string, args []string) {
 		log.Fatalln("error parsing version:", err)
 	}
 
-	addExtRegexp, err := regexp.Compile(*addext)
-	if err != nil {
-		log.Fatalln("error parsing extension inclusion regexp:", err)
+	var addExtRegexp *regexp.Regexp = nil
+	if *addext != "" {
```

Comment:
If we omitted the check against "" we'd end up with a regexp of the empty string, which matches everything: https://play.golang.org/p/_t_Lw5UfxK  I figured that "empty" is a reasonably standard way to describe "unset" so I didn't clarify in the docs; I'm happy to do so if you think it'd help.